ipaplatform/base/paths.py

```diff
@@ -243,8 +244,6 @@ class BasePathNamespace:
     IPA_REPLICA_CONNCHECK = "/usr/sbin/ipa-replica-conncheck"
     IPA_RMKEYTAB = "/usr/sbin/ipa-rmkeytab"
     IPACTL = "/usr/sbin/ipactl"
-    NAMED = "/usr/sbin/named"
-    NAMED_PKCS11 = "/usr/sbin/named-pkcs11"
```

Comment:
Memo to me: We need a way to formerly deprecate constants before we remove them.

Reply:
Good idea!

Reply:
if we consider `ipaplatform` as the part of the public API then I'm wrong.